app/utils/add_sheets_items.py

```diff
@@ -11,314 +10,132 @@
 
-def headsets(sheet):
-    logging.info('Working on the Headsets worksheet')
-    # Get and write thunderbolts
-    bot_sheet = gc.open(sheet).worksheet("Headsets")
-    # get_new_items(sheet, bot_col_value, and_col_value, bot_col, and_col)
-    items = get_new_items(bot_sheet, 'Headsets', 'Headsets', 'C', 'C')
-    cols = {
-        "owner": 'I',
-        "description": 'B',
-        "asset_code": '',
-        'item': ''
-    }
-    # get_new_items_data(sheet, items, cols, item_label)
-    data = get_new_items_data(andela_sheet, items, cols, "Headsets")
-    columns = {
-        1: 'item',
-        2: 'description',
-        3: 'asset_code',
-        4: 'owner'
-    }
-    # first_empty_data_row(sheet, col_value, col_num=1)
-    start_row = first_empty_data_row(bot_sheet, "Headsets")
-    # write_data(sheet, start_row, columns, data, col_len=4)
-    write_data(bot_sheet, start_row, columns, data)
-    logging.info('Finished Working on the Headsets worksheet')
-
-
-def tmacs(sheet):
-    logging.info('Working on the Training Macbooks worksheet')
-    # Get and write thunderbolts
-    bot_sheet = gc.open(sheet).worksheet("Training Macbooks ")
-    # get_new_items(sheet, bot_col_value, and_col_value, bot_col, and_col)
-    items = get_new_items(bot_sheet, 'Training Macbook',
-                          'Training Macbook', 'C', 'C')
-    cols = {
-        "owner": 'I',
-        "description": 'B',
-        "serial": 'G',
-        "asset_code": '',
-        "cohort": 'J',
-        'item': '',
-        'email': ''
-    }
-    # get_new_items_data(sheet, items, cols, item_label)
-    data = get_new_items_data(andela_sheet, items, cols, 'Training Macbook')
-    columns = {
-        1: 'item',
-        2: 'description',
-        3: 'asset_code',
-        4: 'serial',
-        5: 'owner',
-        6: 'email',
-        7: 'cohort'
-    }
-    # first_empty_data_row(sheet, col_value, col_num=1)
-    start_row = first_empty_data_row(bot_sheet, 'Training Macbook')
-    # write_data(sheet, start_row, columns, data, col_len=4)
-    write_data(bot_sheet, start_row, columns, data, col_len=7)
+        for item in filtered_list:
+            col = {
+                "equipement_id": item[2] ,
+                "owner_name": item[8],
+            }
+
+            thunderbolts_list.append(col)
+
+    logging.info('Finished Working on the Thunderbolts worksheet')
+    return thunderbolts_list
+
+def macbooks(sheet_data):
+    logging.info('Working on Macbooks')
+
+    # Get and dump mackbooks
+    # filter_items(sheet, device_type)
+    items  = filter_items(sheet_data, "Company Macbook")
+    items += filter_items(sheet_data, "Staff Macbook")
+    items += filter_items(sheet_data, "D1 Macbook")
+    items += filter_items(sheet_data, "Training Macbook")
+
+    macbook_list = []
+
+    if items:
+        for item in items:
+            col = {
+                "equipment_id": item[2],
+                "serial_number": item[6],
+                "owner_name": item[8],
```

Comment:
In all places where we're grabbing owner_name and owner_cohort, let's strip leading and trailing whitespaces